src/reducers/entities/channels.ts

```diff
@@ -36,7 +38,7 @@ function currentChannelId(state = '', action) {
     }
 }
 
-function channels(state = {}, action) {
+function channels(state: any = {}, action: GenericAction) {
```

Comment:
I think we should have types for this `any`s in the `types/store.ts` file. the types used as "sub-tree" of the `GlobalState` type.